pycroft/lib/membership.py

```diff
@@ -185,3 +189,25 @@ def user_memberships_query(user_id: int, active_groups_only: bool = False):
             .group_by(Membership.id)
     )
     return memberships
+
+
+def change_membership_active_during(
+        membership_id: int,
+        begins_at: datetime,
+        ends_at: Optional[datetime],
+        processor: User
+    ):
+    """
+    modify the active_during field of a user
```

Comment:
`... of a membership`

Reply:
Let's use fancy features to make it a little easier for @go3ranh :]
```suggestion
    modify the active_during field of a membership
 ```